### PR TITLE
Add pprofiler to the prometheus-to-sd.

### PR DESCRIPTION
**Pull request by loburm**
Enable on port 6060 of prometheus-to-sd pprof.

### CODE REVIEW

File: prometheus-to-sd/main.go [hunk 1]
```diff
@@ -78,6 +80,10 @@ func main() {
 	}
 	glog.Infof("GCE config: %+v", gceConf)
 
+	go func() {
+		glog.Infof(http.ListenAndServe(":6060", nil).Error())
```
Comment (crassirostris) [inline prometheus-to-sd/main.go]: I suggest `Fatalf` instead of `Infof`, or at least `Errorf`

Comment (loburm) [inline prometheus-to-sd/main.go]: Done.

File: prometheus-to-sd/main.go [hunk 2]
```diff
@@ -78,6 +80,10 @@ func main() {
 	}
 	glog.Infof("GCE config: %+v", gceConf)
 
+	go func() {
+		glog.Errorf(http.ListenAndServe(":6060", nil).Error())
```
Comment (crassirostris) [inline prometheus-to-sd/main.go]: You can probably do it in a following way:

`glog.Error(http.ListenAndServe(":6060", nil))`

**Comment by crassirostris** [comment]
SGTM

File: prometheus-to-sd/main.go (hunk 2 above)
Comment (loburm) [inline prometheus-to-sd/main.go]: Thanks.

**Comment by crassirostris** [comment]
Just occurred to me. Have you though about two go processes exposing pprof data on this port in the same pod or on the same host with the host networking?

**Comment by loburm** [comment]
Nice catch. Added flag to control exposed port.

**Comment by crassirostris** [comment]
LGTM

**Pull request merged**